docs/3-web-servers/05-server/_samples/app-use/static/index.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Title</title>
+  </head>
+  <body>
+    <a href="./sub/index.html">sub へ</a>
```

Comment:
この教材を見る人が書くコードについては、半角スペースは入れないことになっています。
```suggestion
    <a href="./sub/index.html">subへ</a>
```
で統一することになっています。